components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -190,8 +199,40 @@ std::unique_ptr<RuleIterator> BravePrefProvider::GetRuleIterator(
                                        incognito);
 }
 
+void BravePrefProvider::AddCookieExceptions(ContentSetting content_type,
```

Comment:
this method should take a pattern, making a generic `AddCookieExceptions` still doesn't make sense if it is hard coded to kGoogleOAuthPattern